canonical-web-and-design/cn.ubuntu.com
# new takeover for BQ release in Russia

Done:
New takeover

QA:
Check it visually and check the copydoc.
https://docs.google.com/document/d/172P0ZoiQ5VZ0MIYV38ixK46YwHAmbdHPSmNhOzuent4/edit#

Card:
https://canonical.leankit.com/Boards/View/111185042/118422707
